consent-ui/src/app/common/session.service.ts

```diff
@@ -44,6 +44,17 @@ export class SessionService {
       JSON.parse(sessionStorage.getItem(authorizationId + Session.CONSENT_OBJECT))
     );
   }
+
+  public setXsrfToken(token: string) {
+    console.log('setToken: ', token);
+    sessionStorage.setItem( 'xsrf-token', token);
```

Comment:
Please manage the `xsrf-token` exactly like `redirectCode` (Extend the session enum and use it)